extensions/wikia/PortableInfobox/maintenance/EnableEuropaTheme.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * Enable Europa Theme on wikis that do not use Portable Infoboxes yet
+ * NOT Thread safe! Run only one instance passing arbitrary SERVER_ID, i.e.
+ * SERVER_ID=177 php EnableEuropaTheme.php
+ *
+ * By default runs in dry run mode
+ * Do really enable the rows, use --make-changes option
+ */
+
+require_once( getenv( 'MW_INSTALL_PATH' ) !== false ? getenv( 'MW_INSTALL_PATH' ) . '/maintenance/Maintenance.php' : dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
+
+class EnableEuropaTheme extends Maintenance {
+	public function __construct() {
+		parent::__construct();
+
+		$this->mDescription = 'Enable Europa Theme on wikis that do not use Portable Infoboxes yet';
+		$this->addOption( 'make-changes', 'Do real delete. By default performs a dry run.' );
```

Comment:
Yes, though this is 'copy from clipboard w/ older version there' ;-)
tackled in 17033cd